apps/masterbots.ai/app/b/[id]/page.tsx

```
@@ -12,18 +12,19 @@ export default async function BotThreadsPage({
 }: {
   params: { id: string }
 }) {
+  const chatbotName = (await botNames).get(params.id)
   let chatbot, threads
```

Biome (1.9.4)

[error] Declare variables separately

Unsafe fix: Break out into multiple declarations


(lint/style/useSingleVarDeclarator)